### PR TITLE
[Doc] Add a new metric for managed-ledger add latency

### PR DESCRIPTION
**Pull request by momo-jun**
### Modifications

To align doc with code PR https://github.com/apache/pulsar/pull/4419, make the following changes in documentation.
1. Add a new metric `pulsar_ml_LedgerAddEntryLatencyBuckets`.
2. Differentiate the description for `pulsar_ml_AddEntryLatencyBuckets` and `pulsar_ml_LedgerAddEntryLatencyBuckets`.

> Please kindly note that the same changes will be applied to more historical versions since 2.7.0 as long as it gets approval.

### Preview
image

### Documentation
  
- [x] `doc`

### CODE REVIEW

**Comment by momo-jun** [comment]
Hi @rdhabalia @codelipenghui @hangc0276 @Anonymitaet can you pls review this doc PR?
Please kindly note that the same changes will be applied to more historical versions since 2.7.0 as long as it gets approval.

File: site2/docs/reference-metrics.md [hunk 1]
```diff
@@ -274,14 +274,16 @@ All the managedLedger metrics are labelled with the following labels:
...
 | pulsar_ml_LedgerSwitchLatencyBuckets_OVERFLOW | Gauge | The ledger switch latency > 1s |
+| pulsar_ml_LedgerAddEntryLatencyBuckets | Histogram | The latency to persist entry into a ledger with a given quantile (threshold). <br /> Available quantile: <br /><ul><li> quantile="0.0_0.5" is AddEntryLatency between (0.0ms, 0.5ms]</li> <li>quantile="0.5_1.0" is AddEntryLatency between (0.5ms, 1.0ms]</li><li>quantile="1.0_5.0" is AddEntryLatency between (1ms, 5ms]</li><li>quantile="5.0_10.0" is AddEntryLatency between (5ms, 10ms]</li><li>quantile="10.0_20.0" is AddEntryLatency between (10ms, 20ms]</li><li>quantile="20.0_50.0" is AddEntryLatency between (20ms, 50ms]</li><li>quantile="50.0_100.0" is AddEntryLatency between (50ms, 100ms]</li><li>quantile="100.0_200.0" is AddEntryLatency between (100ms, 200ms]</li><li>quantile="200.0_1000.0" is AddEntryLatency between (200ms, 1s]</li></ul>|
+| pulsar_ml_LedgerAddEntryLatencyBuckets_OVERFLOW | Gauge | The latency to persist entry into a ledger > 1s |
```
Comment (Anonymitaet) [inline site2/docs/reference-metrics.md:286]:
```suggestion
| pulsar_ml_LedgerAddEntryLatencyBuckets_OVERFLOW | Gauge | The latency to persist an entry into a ledger. The overflow is triggered if the latency is longer than 1 second. |
```

Do you mean this? @hangc0276 @momo-jun

Comment (hangc0276) [reply site2/docs/reference-metrics.md:286]: The latency to persist an entry into a ledger. The overflow is triggered if the latency is longer than 1 second.
-----
The number of times the write latency is longer than 1 second

File: site2/docs/reference-metrics.md [hunk 2]
```diff
@@ -274,14 +274,16 @@ All the managedLedger metrics are labelled with the following labels:
...
 | pulsar_ml_LedgerSwitchLatencyBuckets_OVERFLOW | Gauge | The ledger switch latency > 1s |
+| pulsar_ml_LedgerAddEntryLatencyBuckets | Histogram | The latency to persist entry into a ledger with a given quantile (threshold). <br /> Available quantile: <br /><ul><li> quantile="0.0_0.5" is AddEntryLatency between (0.0ms, 0.5ms]</li> <li>quantile="0.5_1.0" is AddEntryLatency between (0.5ms, 1.0ms]</li><li>quantile="1.0_5.0" is AddEntryLatency between (1ms, 5ms]</li><li>quantile="5.0_10.0" is AddEntryLatency between (5ms, 10ms]</li><li>quantile="10.0_20.0" is AddEntryLatency between (10ms, 20ms]</li><li>quantile="20.0_50.0" is AddEntryLatency between (20ms, 50ms]</li><li>quantile="50.0_100.0" is AddEntryLatency between (50ms, 100ms]</li><li>quantile="100.0_200.0" is AddEntryLatency between (100ms, 200ms]</li><li>quantile="200.0_1000.0" is AddEntryLatency between (200ms, 1s]</li></ul>|
```
Comment (hangc0276) [inline site2/docs/reference-metrics.md:285]: The latency to persist entry by bookie client from broker to BookKeeper service with a given quantile (threshold).

Comment (momo-jun) [reply site2/docs/reference-metrics.md]: Thanks for your comments. I've updated the description. PTAL.

File: site2/docs/reference-metrics.md [hunk 3]
```diff
@@ -274,14 +274,16 @@ All the managedLedger metrics are labelled with the following labels:
...
-| pulsar_ml_AddEntryLatencyBuckets_OVERFLOW | Gauge | The add entry latency > 1s |
+| pulsar_ml_AddEntryLatencyBuckets | Histogram | The add entry latency of a ledger with a given quantile (threshold), including time spent on waiting in queue on the broker side.<br> Available quantile: <br><ul><li> quantile="0.0_0.5" is AddEntryLatency between (0.0ms, 0.5ms]</li> <li>quantile="0.5_1.0" is AddEntryLatency between (0.5ms, 1.0ms]</li><li>quantile="1.0_5.0" is AddEntryLatency between (1ms, 5ms]</li><li>quantile="5.0_10.0" is AddEntryLatency between (5ms, 10ms]</li><li>quantile="10.0_20.0" is AddEntryLatency between (10ms, 20ms]</li><li>quantile="20.0_50.0" is AddEntryLatency between (20ms, 50ms]</li><li>quantile="50.0_100.0" is AddEntryLatency between (50ms, 100ms]</li><li>quantile="100.0_200.0" is AddEntryLatency between (100ms, 200ms]</li><li>quantile="200.0_1000.0" is AddEntryLatency between (200ms, 1s]</li></ul>|
```
Comment (Anonymitaet) [inline site2/docs/reference-metrics.md:277]:
```suggestion
| pulsar_ml_AddEntryLatencyBuckets | Histogram | The added entry latency of a ledger with a given quantile (threshold), including time spent on waiting in queue on the broker side.<br> Available quantile: <br><ul><li> quantile="0.0_0.5" is AddEntryLatency between (0.0ms, 0.5ms]</li> <li>quantile="0.5_1.0" is AddEntryLatency between (0.5ms, 1.0ms]</li><li>quantile="1.0_5.0" is AddEntryLatency between (1ms, 5ms]</li><li>quantile="5.0_10.0" is AddEntryLatency between (5ms, 10ms]</li><li>quantile="10.0_20.0" is AddEntryLatency between (10ms, 20ms]</li><li>quantile="20.0_50.0" is AddEntryLatency between (20ms, 50ms]</li><li>quantile="50.0_100.0" is AddEntryLatency between (50ms, 100ms]</li><li>quantile="100.0_200.0" is AddEntryLatency between (100ms, 200ms]</li><li>quantile="200.0_1000.0" is AddEntryLatency between (200ms, 1s]</li></ul>|
```

**Comment by momo-jun** [comment]
Hi @hangc0276 @Anonymitaet I've streamlined more descriptions. PTAL.

**Review by Anonymitaet — APPROVED** [review]

**Review by hangc0276 — APPROVED** [review]

**Pull request merged**